src/utils/url_params.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * decodeUrlParams - converts a keyvalue object into URLEncoded params
+ * @param {object} - object of key and its corresponding values
+ * @returns {string} - urlencoded query string
+ */
+const decodeUrlParams = (url, splitString = "&", paramChar = "?") => {
+    console.log("decodeUrlParams");
+    console.log(url);
+    const hashAt = url.indexOf(paramChar);
+    const engagementQueryArgs = hashAt >= 0 && decodeURIComponent(url.substr(hashAt+1));
+    console.log(engagementQueryArgs);
+    return parseQueryArgs(engagementQueryArgs, splitString);
```

Comment:
do the forward declarations of const actually work? 
i would simply use `function` declarations instead of `const`
you can also say directly: `export function decordUrlParams(...) {`